source/team.html.erb

```diff
@@ -54,7 +54,7 @@ responsive: true
     The Ember CLI Team
   </h2>
   <p class="lead">
-    
+  The Ember CLI Team is responsible for (responsibilities). The team meets on (day), and the public notes can be found <a href="github.com/ember-cli">on Github</a>.
```

Comment:
Ember CLI is the command line interface for managing and packaging Ember.js
applications.

Reply:
the public notes can be found at [https://github.com/emberjs/core-notes/tree/master/ember-cli](https://github.com/emberjs/core-notes/tree/master/ember-cli)

Reply:
The team meets on Thursdays at 4pm Eastern Time, using a bluejeans (link is put into the #dev-ember-cli 10 minutes before). They are open to the public